src/test/java/uk/gov/hmcts/darts/retention/service/impl/RetentionGetPolicyTypesServiceImplTest.java

```diff
@@ -77,6 +80,7 @@ private void setupStubs() {
             authorisationApi,
             policyDurationValidator,
             createPolicyTypeValidator,
+            editPolicyTypeValidator,
```

Comment:
We could also change the tests to validate that the mockito validator object is being called.

Reply:
I have now added `verifyNoInteractions(editPolicyTypeValidator)` to each test in here - the test is concerned only with "getting" the policies, not editing them.